src/utils/voice_recorder.py

```
@@ -0,0 +1,187 @@
+
+    async def _auto_disconnect(self, max_duration: int) -> None:
+        """自動斷開連接
+
+        Args:
+            max_duration: 最長連接時間（秒）
+        """
+        try:
+            await asyncio.sleep(max_duration)
+            if self.is_connected:
+                await self.disconnect_from_voice()
+                logger.info(f"連接達到最長時間 {max_duration} 秒，自動斷開")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)